Templates/Integrations/AnalyticsAndSIEM/AnalyticsAndSIEM.py

```diff
@@ -0,0 +1,354 @@
+
+        Returns:
+            Response from API
+        """
+        # The service endpoint to request from
+        suffix = 'event'
+        # Dictionary of params for the request
+        params = assign_params(eventId=event_id)
+        # Send a request using our http_request wrapper
+        return self._http_request('DELETE', suffix, params=params)
+
+    def update_event_request(self, event_id: AnyStr, description: Optional[AnyStr] = None,
+                             assignee: Optional[List[str]] = None) -> Dict:
+        """Update given event
+
+        Args:
+            description: change description of event
+            assignee: User to assign event to
+            event_id: event ID
+
+        Returns:
+            Response from API
+        """
+        suffix = 'event'
+        params = assign_params(eventId=event_id, description=description, assignee=assignee)
+        return self._http_request('POST', suffix, params=params)
+
+    def create_event_request(self, description: str, assignee: List[str] = None) -> Dict:
+        """Creates given event.
+
+        Args:
+            description: change description of event
+            assignee: User to assign event to
+
+        Returns:
+            Response from API
+        """
+        suffix = 'event'
+        params = assign_params(description=description, assignee=assignee)
+        return self._http_request('POST', suffix, params=params)
+
+    def query_request(self, **kwargs):
+        suffix = 'query'
+        return self._http_request('GET', suffix, params=kwargs)
+
+
+''' HELPER FUNCTIONS '''
+
+
+def build_dict(event: Dict) -> Dict:
+    """Builds Dict formatted for Demisto
+
+    Args:
+        event: One event from API
+
+    Returns:
+        Dict formatted for Demisto
+    """
+    return {
+        'ID': event.get('eventId'),
+        'Description': event.get('description'),
+        'Created': event.get('createdAt'),
+        'IsActive': event.get('isActive'),
+        'Assignee': [
+            {
+                'Name': user.get('name'),
+                'ID': user.get('id')
+            } for user in event.get('assignee', [])
+        ]
+    }
+
+
+def build_context(events: Union[Dict, List]) -> Union[Dict, List]:
+    """Formatting API response to Demisto Context
+
+    Args:
+        events: Raw response from API call, can be List or Dict
+
+    Returns:
+        formatted Dict/List (according to the input type)
+    """
+    if isinstance(events, list):
+        return [build_dict(event) for event in events]
+    return build_dict(events)
+
+
+''' COMMANDS '''
+
+
+def test_module(client: Client) -> Tuple[str, Dict, Dict]:
+    """Performs basic get request to see if the API is reachable and authentication works.
+    """
+    results = client.test_module_request()
+    if 'version' in results:
+        return 'ok', {}, {}
+    raise DemistoException('Test module failed, {}'.format(results))
+
+
+def fetch_incidents(client: Client):
```

Comment:
pass the `last_run` object like you do with args in rest of the commands.
return the incidents and new `last_run` object